engine/src/main/java/org/pentaho/di/trans/steps/textfileoutput/TextFileFieldDTO.java

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2024 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
As this file is added now, just 2024 would suffice Vamsi.

Reply:
Resolved